### ISSUE TITLE
Native chooser for ios

### ISSUE THREAD

**Issue by jahicDario** [dpa99c/react-native-launch-navigator#8]
**I'm submitting a ...**  (check one with "x"):
- [ ] bug report
- [x] feature request
- [ ] documentation issue

# Feature request

Is there a way to let the user to pick the app to start navigation like in Android with LaunchNavigator.APP.GEO. 
I noticed that you have that option for phonegap: _launchnavigator.APP.USER_SELECT (Android & iOS) - invokes native UI for user to select available navigation app_
and I think it would be nice to implement this for ReactNative as well.

**Comment by dpa99c** [comment]
Yes this is something I'd like to port across from the Cordova/Phonegap version when I have some time.

**Comment by ansarikhurshid786** [comment]
it has native chooser for android 
 LaunchNavigator.navigate([this.state.onwaniMarker[1], this.state.onwaniMarker[0]], {
                    app: LaunchNavigator.APP.GEO
                }).then(() => {
                    console.log("Launched navigator");
                });

**Comment by ansarikhurshid786** [comment]
is there any option for ios?
